covfee/server/rest_api/journeys.py

```diff
@@ -81,3 +81,12 @@ def node_ready(jid, nidx, value):
     socketio.emit("status", payload, namespace="/admin")
 
     return "", 200
+
+# Return the annotator_data for the annotator working on this journey
+@api.route("/journeys/<jid>/annotator_data")
+@admin_required
+def annotator_data(jid):
+    res :JourneyInstance  = app.session.query(JourneyInstance).get(bytes.fromhex(jid))
+    if res.annotator is None:
+        return {}
```

Comment:
does `return {}` creates a `null` payload in the frontend ?

Reply:
It shows as `{}` in typescript, how do I check for that?

Reply:
I tried with ce37882 , not sure if it makes sense. I tried `=== null` and `=== undefined` but those didn't work.

Reply:
Remove the checks for the prolific and date attributes